app/assets/stylesheets/views/_worldwide-organisation.scss

```diff
@@ -58,3 +58,20 @@
     }
   }
 }
+
+.corporate-information__sub-navigation {
```

Comment:
I'm always slightly unnerved by css changes. Were these copied over from Whitehall or i this in place to make it look the same as WH? Basically just want to make sure there's not already a component that would cover this styling

Reply:
Might be worth looking at https://components.publishing.service.gov.uk/component-guide/list or https://design-system.service.gov.uk/styles/typography/#lists, which it is based on.

Reply:
That change has been merged already. This was branching off another branch. Have rebased.